lib/percona_migrator/alter_argument.rb

```diff
@@ -1,30 +1,32 @@
 module PerconaMigrator
+  class InvalidAlterStatement < StandardError; end
 
   # Represents the '--alter' argument of Percona's pt-online-schema-change
   # See https://www.percona.com/doc/percona-toolkit/2.0/pt-online-schema-change.html
   class AlterArgument
     ALTER_TABLE_REGEX = /ALTER TABLE `(\w+)` /
```

Comment:
/\AALTER TABLE `(\w+)` / por favor :pray: 
http://batsov.com/articles/2013/12/04/regexp-anchors-in-ruby/